APMrover2/APMrover2.cpp

```diff
@@ -434,9 +434,15 @@ void Rover::update_current_mode(void)
     switch (control_mode){
     case AUTO:
     case RTL:
-        calc_lateral_acceleration();
-        calc_nav_steer();
-        calc_throttle(g.speed_cruise);
+        if (!do_auto_rotation) {
+            calc_lateral_acceleration();
+            calc_nav_steer();
+            calc_throttle(g.speed_cruise);
+        }
+        else {
```

Comment:
http://ardupilot.org/dev/docs/style-guide.html#else-statement
I think that it was better if possible.